Task6.1/Book.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace BookCatalogTask
+{
+    public class Book
+    {
+        private string _title;
+
+        public string Title
+        {
+            get => _title;
+            set
+            {
+                if (value == null)
+                {
+                    throw new ArgumentNullException(nameof(Title), "Title cannot be null.");
+                }
+
+                _title = value;
+            }
+        }
+
+        public DateTime? PublicationDate { get; set; }
+        public List<Author> Authors { get; set; }

```

Comment:
Book has to check ISBN by himself, not to let anybody to do this. Think that book with wrong isbn is not a valid object

Reply:
Fixed :)